src/Guard.php

```diff
@@ -231,38 +231,7 @@ protected function validateToken($name, $value)
      */
     protected function createToken()
     {
-        $token = "";
-
-        if (function_exists("random_bytes")) {
-            $rawToken = random_bytes($this->strength);
-            if ($rawToken !== false) {
```

Comment:
`$rawToken == false` [can not happen](http://php.net/manual/en/function.random-bytes.php#refsect1-function.random-bytes-returnvalues), obviously a copy-paste-mistake of the [`openssl_random_pseudo_bytes` branch](http://php.net/manual/de/function.openssl-random-pseudo-bytes.php#refsect1-function.openssl-random-pseudo-bytes-returnvalues).